NoteFind/View Controllers/LoginViewController.m

```diff
@@ -26,18 +32,21 @@ - (IBAction)didTapLogin:(id)sender {
     
     [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
         if (error != nil) {
-            NSLog(@"User log in failed: %@", error.localizedDescription);
-            [self incorrectLoginInfo];
+            if ([username isEqual:@""] || [password isEqual:@""]) {
```

Comment:
- For string equality, prefer using `isEqualToString` when you know you're comparing strings. https://bignerdranch.com/blog/isequal-vs-isequaltostring/
- This just checks if the string is empty, do we need to validate if the user types in a bunch of spaces?